about.html

```diff
@@ -9,55 +9,80 @@
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
 
+<div>
+  <nav> 
+    <ul id= "ul_nav">
+        <li><img class="logo" src="img/lambda-black.png" alt="I <3 Lambda."></li>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="#">Products</a></li>
+        <li><a href="#">Blog</a></li>
+        <li><a href="#">Contact</a></li>
+   </ul>
+  </nav>
+</div>
+
+<div id="header_image"><img src="img/jumbo-about.png"></div>
+
 </head>
 
+
+
 <body>
-    <div class="container about-page">
+    
+<div class="container about-page">
      
     About
 
     Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
```

Comment:
`About` and `Leverage agile...` should be in some sort of element, like h2 and p